packages/EasyErrorHandler/src/Bridge/Symfony/Builder/ApiPlatformValidationExceptionBuilder.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Builder;
+
+use ApiPlatform\Core\Bridge\Symfony\Validator\Exception\ValidationException;
+use EonX\EasyErrorHandler\Builders\AbstractErrorResponseBuilder;
+use EonX\EasyErrorHandler\Interfaces\TranslatorInterface;
+use Throwable;
+
+final class ApiPlatformValidationExceptionBuilder extends AbstractErrorResponseBuilder
+{
+    /**
+     * @var string[]
+     */
+    private $keys;
+
+    /**
+     * @var \EonX\EasyErrorHandler\Interfaces\TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @param null|mixed[] $keys
+     */
+    public function __construct(TranslatorInterface $translator, ?array $keys = null, ?int $priority = null)
+    {
+        $this->translator = $translator;
+        $this->keys = $keys ?? [];
+
+        parent::__construct($priority);
+    }
+
+    /**
+     * @param mixed[] $data
+     *
+     * @return mixed[]
+     */
+    public function buildData(Throwable $throwable, array $data): array
+    {
+        if ($throwable instanceof ValidationException) {
+            $violations = [];
+
+            foreach ($throwable->getConstraintViolationList() as $violation) {
+                $propertyPath = $violation->getPropertyPath();
+
+                if (\array_key_exists($propertyPath, $violations) === false) {
+                    $violations[$propertyPath] = [];
+                }
+
+                $violations[$propertyPath][] = $violation->getMessage();
+            }
+
+            $exceptionKey = $this->getKey('exception');
+            $exceptionMessageKey = $this->getKey('extended_exception_keys.message');
+
+            if (\is_array($data[$exceptionKey] ?? null)) {
+                $data[$exceptionKey][$exceptionMessageKey] = 'Entity validation failed.';
```

Comment:
Not every project might want to have this generic message `Entity validation failed.`, can we please make it configurable.

Reply:
Actually it could just be a translation.

Reply:
I'm not sure that it has to be a translatable. It's a message for the exception and it won't be visible for a user on production. Anyway, do you have any suggestions for the translation key?

Reply:
@itorgov we already have the translation key: `extended_exception_keys.message`

Reply:
Done by using another key :-)